### PR TITLE
Misc fixes to remove unsupported features and fix style.

### PR DESCRIPTION
**Pull request by stbof**

### CODE REVIEW

File: v22.1/reset-storage-parameter.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 ---
 
-The `RESET (storage parameter)` [statement](sql-statements.html) reverts the value of a storage parameter on a table or an index to its default value.
+The `RESET (storage parameter)` [statement](sql-statements.html) reverts the value of a storage parameter on a table to its default value.
 
 {{site.data.alerts.callout_info}}
-The `SET (storage parameter)` is a subcommand of [`ALTER TABLE`](alter-table.html) and [`ALTER INDEX`](alter-index.html).
+The `RESET (storage parameter)` is a subcommand of [`ALTER TABLE`](alter-table.html).
+
+To reset a storage parameter on an existing index, you must drop and [recreate the index with the storage parameter](with-storage-parameter.html).
```
Comment (nickvigilante) [inline v22.1/reset-storage-parameter.md:13]:
```suggestion
To reset a storage parameter on an existing index, you must drop and [recreate the index without the storage parameter](with-storage-parameter.html).
```

Isn't resetting the storage parameter removing the storage parameter?

File: v22.1/spatial-indexes.md [hunk 2]
```diff
@@ -50,25 +48,25 @@ CockroachDB uses the "divide the space" approach for the following reasons:
 - It requires no balancing operations, unlike [R-tree indexes](https://en.wikipedia.org/wiki/R-tree).
 - Inserts under this approach require no locking.
 - Bulk ingest is simpler to implement than under other approaches.
-- It allows advanced users to make a per-object tradeoff between index size and false positives during index creation. (See [Tuning spatial indexes](#tuning-spatial-indexes) below.)
```
Comment (nickvigilante) [inline v22.1/spatial-indexes.md:53]: Not a blocker, but would it make sense to add a Vale linter that warns about these positional prepositions? If so, I can take that as a to-do.

File: v22.2/reset-storage-parameter.md [hunk 3]
```diff
@@ -13,10 +13,12 @@ docs_area: reference.sql
...
 {% endif %}
 
-The `RESET (storage parameter)` [statement](sql-statements.html) reverts the value of a storage parameter on a table or an index to its default value.
+The `RESET (storage parameter)` [statement](sql-statements.html) reverts the value of a storage parameter on a table to its default value.
 
 {{site.data.alerts.callout_info}}
-The `SET (storage parameter)` is a subcommand of [`ALTER TABLE`](alter-table.html) and [`ALTER INDEX`](alter-index.html).
+The `RESET (storage parameter)` is a subcommand of [`ALTER TABLE`](alter-table.html).
+
+To reset a storage parameter on an existing index, you must drop and [recreate the index with the storage parameter](with-storage-parameter.html).
```
Comment (nickvigilante) [inline v22.2/reset-storage-parameter.md:21]:
```suggestion
To reset a storage parameter on an existing index, you must drop and [recreate the index without the storage parameter](with-storage-parameter.html).
```

Isn't resetting the storage parameter removing the storage parameter?

File: v22.2/set-storage-parameter.md [hunk 4]
```diff
@@ -55,7 +53,7 @@ The user must be a member of the [`admin`](security-reference/authorization.html
...
+{% include_cached new-in.html version="v22.1" %} In some situations, you may want to exclude a table's row data from a [backup](backup.html). For example, you have a table that contains high-churn data that you would like to [garbage collect](architecture/storage-layer.html#garbage-collection) more quickly than the [incremental backup](take-full-and-incremental-backups.html#incremental-backups) schedule for the database or cluster holding the table. You can use the `exclude_data_from_backup = true` parameter with a [`CREATE TABLE`](create-table.html#create-a-table-with-data-excluded-from-backup) or `ALTER TABLE` statement to mark a table's row data for exclusion from a backup.
```
Comment (nickvigilante) [inline v22.2/set-storage-parameter.md:56]:
```suggestion
In some situations, you may want to exclude a table's row data from a [backup](backup.html). For example, you have a table that contains high-churn data that you would like to [garbage collect](architecture/storage-layer.html#garbage-collection) more quickly than the [incremental backup](take-full-and-incremental-backups.html#incremental-backups) schedule for the database or cluster holding the table. You can use the `exclude_data_from_backup = true` parameter with a [`CREATE TABLE`](create-table.html#create-a-table-with-data-excluded-from-backup) or `ALTER TABLE` statement to mark a table's row data for exclusion from a backup.
```

File: v22.2/spatial-indexes.md [hunk 5]
```diff
@@ -50,25 +48,25 @@ CockroachDB uses the "divide the space" approach for the following reasons:
...
+Next, let's look at a 3-dimensional image that shows the cube and sphere more clearly. Each cube face is mapped to the quadtree data structure mentioned, and each node in the quadtree is numbered using a [Hilbert space-filling curve](https://en.wikipedia.org/wiki/Hilbert_curve) which preserves locality of reference. In the following image, you can imagine the points of the Hilbert curve on the rear face of the cube being projected onto the sphere in the center. The use of a space-filling curve means that two shapes that are near each other on the sphere are very likely to be near each other on the line that makes up the Hilbert curve. This is good for performance.
 
-<img style="display: block; margin-left: auto; margin-right: auto; width: 50%" src="{{ 'images/v22.2/geospatial/s2-cubed-sphere-3d.png' | relative_url }}" alt="S2 Cubed Sphere - 3D">
+<img style="display: block; margin-left: auto; margin-right: auto; width: 50%" src="{{ 'images/v22.1/geospatial/s2-cubed-sphere-3d.png' | relative_url }}" alt="S2 Cubed Sphere - 3D">
```
Comment (nickvigilante) [inline v22.2/spatial-indexes.md:67]:
```suggestion
<img style="display: block; margin-left: auto; margin-right: auto; width: 50%" src="{{ 'images/v22.2/geospatial/s2-cubed-sphere-3d.png' | relative_url }}" alt="S2 Cubed Sphere - 3D">
```

File: v22.2/spatial-indexes.md [hunk 6]
```diff
@@ -50,25 +48,25 @@ CockroachDB uses the "divide the space" approach for the following reasons:
...
+Under the hood, CockroachDB uses the [S2 geometry library](https://s2geometry.io/) to divide the space being indexed into a [quadtree](https://en.wikipedia.org/wiki/Quadtree) data structure with a set number of levels and a data-independent shape. Each node in the quadtree (really, [S2 cell](https://s2geometry.io/devguide/s2cell_hierarchy.html)) represents some part of the indexed space and is divided once horizontally and once vertically to produce 4 child cells in the next level. The following image shows visually how a location (marked in red) is represented using levels of a quadtree:
 
-<img style="display: block; margin-left: auto; margin-right: auto; width: 50%" src="{{ 'images/v22.2/geospatial/quadtree.png' | relative_url }}" alt="Quadtree">
+<img style="display: block; margin-left: auto; margin-right: auto; width: 50%" src="{{ 'images/v22.1/geospatial/quadtree.png' | relative_url }}" alt="Quadtree">
```
Comment (nickvigilante) [inline v22.2/spatial-indexes.md:59]:
```suggestion
<img style="display: block; margin-left: auto; margin-right: auto; width: 50%" src="{{ 'images/v22.2/geospatial/quadtree.png' | relative_url }}" alt="Quadtree">
```

File: v22.2/spatial-indexes.md [hunk 7]
```diff
@@ -50,25 +48,25 @@ CockroachDB uses the "divide the space" approach for the following reasons:
...
+Visually, you can think of the S2 library as enclosing a sphere in a cube. We map from points on each face of the cube to points on the face of the sphere. As you can see in the following 2-dimensional picture, there is a projection that occurs in this mapping: the lines entering from the left mark points on the cube face, and are "refracted" by the material of the cube face before touching the surface of the sphere. This projection reduces the distortion that would occur if the points on the cube face were projected straight onto the sphere.
 
-<img style="display: block; margin-left: auto; margin-right: auto; width: 50%" src="{{ 'images/v22.2/geospatial/s2-cubed-sphere-2d.png' | relative_url }}" alt="S2 Cubed Sphere - 2D">
+<img style="display: block; margin-left: auto; margin-right: auto; width: 50%" src="{{ 'images/v22.1/geospatial/s2-cubed-sphere-2d.png' | relative_url }}" alt="S2 Cubed Sphere - 2D">
```
Comment (nickvigilante) [inline v22.2/spatial-indexes.md:63]:
```suggestion
<img style="display: block; margin-left: auto; margin-right: auto; width: 50%" src="{{ 'images/v22.2/geospatial/s2-cubed-sphere-2d.png' | relative_url }}" alt="S2 Cubed Sphere - 2D">
```

File: v22.2/spatial-indexes.md [hunk 8]
```diff
@@ -99,20 +97,20 @@ We will generate coverings for the following geometry object, which describes a
 'SRID=4326;LINESTRING(-76.4955 42.4405,  -75.6608 41.4102,-73.5422 41.052, -73.929 41.707, -76.4955 42.4405)'
 ~~~
 
-The animated image below shows the S2 coverings that are generated as we increase the `s2_max_cells` parameter (described below) from the 1 to 30 (minimum to maximum):
+The animated following image shows the S2 coverings that are generated as we increase the `s2_max_cells` parameter from the 1 to 30 (minimum to maximum):
 
-<img style="display: block; margin-left: auto; margin-right: auto; width: 50%" src="{{ 'images/v22.2/geospatial/s2-coverings.gif' | relative_url }}" alt="Animated GIF of S2 Coverings - Levels 1 to 30">
+<img style="display: block; margin-left: auto; margin-right: auto; width: 50%" src="{{ 'images/v22.1/geospatial/s2-coverings.gif' | relative_url }}" alt="Animated GIF of S2 Coverings - Levels 1 to 30">
```
Comment (nickvigilante) [inline v22.2/spatial-indexes.md:102]:
```suggestion
<img style="display: block; margin-left: auto; margin-right: auto; width: 50%" src="{{ 'images/v22.2/geospatial/s2-coverings.gif' | relative_url }}" alt="Animated GIF of S2 Coverings - Levels 1 to 30">
```

File: _includes/v22.2/misc/table-storage-parameters.md [hunk 9]
```diff
@@ -1,21 +1,19 @@
 | Parameter name      | Description | Data type | Default value |
 |---------------------+----------------------+-----+------|
-| `exclude_data_from_backup` | Excludes the data in this table from any future backups. | Boolean | `false` |
+| `exclude_data_from_backup` | **New in v22.1:** Excludes the data in this table from any future backups. | Boolean | `false` |
```
Comment (nickvigilante) [inline _includes/v22.2/misc/table-storage-parameters.md:3]:
```suggestion
| `exclude_data_from_backup` | Excludes the data in this table from any future backups. | Boolean | `false` |
```

File: v22.2/spatial-indexes.md [hunk 10]
```diff
@@ -99,20 +97,20 @@ We will generate coverings for the following geometry object, which describes a
...
 
-Below are the same images presented in a grid. You can see that as we turn up the `s2_max_cells` parameter, more work is done by CockroachDB to discover a tighter and tighter covering (that is, a covering using more and smaller cells). The covering for this particular shape reaches a reasonable level of accuracy when `s2_max_cells` reaches 10, and stops improving much past 12.
+Here are the same images presented in a grid. You can see that as we turn up the `s2_max_cells` parameter, more work is done by CockroachDB to discover a tighter and tighter covering (that is, a covering using more and smaller cells). The covering for this particular shape reaches a reasonable level of accuracy when `s2_max_cells` reaches 10, and stops improving much past 12.
 
-<img style="display: block; margin-left: auto; margin-right: auto; width: 50%" src="{{ 'images/v22.2/geospatial/s2-coverings-tiled.png' | relative_url }}" alt="Static image of S2 Coverings - Levels 1 to 30">
+<img style="display: block; margin-left: auto; margin-right: auto; width: 50%" src="{{ 'images/v22.1/geospatial/s2-coverings-tiled.png' | relative_url }}" alt="Static image of S2 Coverings - Levels 1 to 30">
```
Comment (nickvigilante) [inline v22.2/spatial-indexes.md:106]:
```suggestion
<img style="display: block; margin-left: auto; margin-right: auto; width: 50%" src="{{ 'images/v22.2/geospatial/s2-coverings-tiled.png' | relative_url }}" alt="Static image of S2 Coverings - Levels 1 to 30">
```

File: v22.2/spatial-indexes.md [hunk 11]
```diff
@@ -148,26 +147,22 @@ SELECT ST_AsGeoJSON(st_collect(geom)) FROM tmp_viz;
...
 ~~~
 
-When you paste the JSON output above into [geojson.io](http://geojson.io), it generates the picture below, which shows both the `LINESTRING` and its S2 covering based on the options you passed to `st_s2covering`.
+When you paste the JSON output into [geojson.io](http://geojson.io), it generates the following picture, which shows both the `LINESTRING` and its S2 covering based on the options you passed to `st_s2covering`.
 
-<img style="display: block; margin-left: auto; margin-right: auto; width: 50%" src="{{ 'images/v22.2/geospatial/s2-linestring-example-covering.png' | relative_url }}" alt="S2 LINESTRING example covering">
+<img style="display: block; margin-left: auto; margin-right: auto; width: 50%" src="{{ 'images/v22.1/geospatial/s2-linestring-example-covering.png' | relative_url }}" alt="S2 LINESTRING example covering">
```
Comment (nickvigilante) [inline v22.2/spatial-indexes.md:152]:
```suggestion
<img style="display: block; margin-left: auto; margin-right: auto; width: 50%" src="{{ 'images/v22.2/geospatial/s2-linestring-example-covering.png' | relative_url }}" alt="S2 LINESTRING example covering">
```

**Review by nickvigilante — CHANGES_REQUESTED** [review]
A few nits:

- I think copypasta overwrote the version-specific image tags in the 22.2 docs, so I added suggestions where I found them to put back 22.2.
- Some language around resetting a storage parameter
- Aside question: should we add a Vale linter to strip out certain positional prepositions like above/below and other directional words such as top/bottom and left/right?

Otherwise, LGTM!

File: v22.1/spatial-indexes.md (hunk 2 above)
Comment (stbof) [reply v22.1/spatial-indexes.md:53]: I think below, above etc aren't always positional, so I think a linter rule would throw too many false +.  I'm hoping that education writing for accessibility will help.

**Review by nickvigilante — APPROVED** [review]
LGTM!

**Pull request merged**